src/cond.d

```diff
@@ -89,18 +89,63 @@ extern (C++) class DVCondition : Condition
  */
 extern (C++) final class DebugCondition : DVCondition
 {
+    /**
+     * Set the global version level
```

Comment:
Actually it should be global 'debug' level, copy pasta error.
What it is should be on the variable, not its setter though.